oci/oci_linux.go

```diff
@@ -0,0 +1,57 @@
+// +build linux
+
+package oci
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+
+	"github.com/containerd/cgroups"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"golang.org/x/sys/unix"
+)
+
+func getExitCode(err error) int32 {
+	if exitErr, ok := err.(*exec.ExitError); ok {
+		if status, ok := exitErr.Sys().(unix.WaitStatus); ok {
+			return int32(status.ExitStatus())
+		}
+	}
+	return -1
+}
+
+func createContainerPlatform(c *Container, cgroupParent string, pid int) error {
+	control, err := cgroups.New(cgroups.V1, cgroups.StaticPath(filepath.Join(cgroupParent, "/crio-conmon-"+c.id)), &rspec.LinuxResources{})
+	if err != nil {
+		return fmt.Errorf("Failed to add conmon to cgroupfs sandbox cgroup: %v", err)
+	} else {
```

Comment:
Lint [wants you to flatten this `else`][1].

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/349956654#L1120

Reply:
hmm. my lint doesn't show that ...